lib/ansible/modules/packaging/language/pip.py

```diff
@@ -250,7 +250,9 @@ def _get_full_name(name, version=None):
     if version is None or version == "":
         resp = name
     else:
-        resp = name + '==' + version
+        if version[0].isdigit():
+            version = "==%s" % version
+        resp = "\"%s (%s)\"" % (name, version)
```

Comment:
This works for new installs? What is this new format used for?

Reply:
I'm not sure whether this is compatible with all pip version... I will have more tests.

Reply:
FTR It's incompatible with outdated versions of pip